packages.go

```diff
@@ -9,15 +9,99 @@ import (
 	"time"
 )
 
+type screeningPackage string
+
+// ScreeningPackage ...
+type ScreeningPackage interface {
+	Slug() string
+}
+
+var packages = make(map[ScreeningPackage]string)
+
+// Code returns Package title string
+func (a screeningPackage) Slug() string {
+	return string(a)
+}
+
+// Screening Packages
+const (
```

Comment:
@chandradeepak let's be consistent with the CamelCase. Example, `SsnScreen` == `SSNScreen`, `Samuraishimmer` == `SamuraiShimmer`, etc.